haas/api.py

```diff
@@ -423,6 +423,39 @@ def node_detach_network(node, nic, network):
     return '', 202
 
 
+@rest_call('PUT', '/TPM_metadata/<tpm_metadata>', Schema({
```

Comment:
Could we make the API more generic?
For example, instead of embedding TPM in the URL, make the PUT go to: `/node/<node>/metadata` with a body of
```json
{
"key": key,
"value": value
}
```
In the documentation, we'd then document the "standardized" metadata pieces, like "TPM_EK" and "TPM_BASE_PCRs".